image_detector/LineSegment[] lines
